types/office-js-preview/index.d.ts

```diff
@@ -89713,12 +89936,52 @@ declare namespace Word {
          */
         readonly result: Word.Range;
         /**
-         * Gets the field's code instruction.
+         * Gets or sets the field's code instruction.
          *
          * @remarks
          * [Api set: WordApi 1.4]
          */
-        readonly code: string;
+        code: string;
+        /**
+         * Gets or sets data in an "Addin" field. If the field isn't an "Addin" field, it is null and it will throw a general exception when setting it.
```

Comment:
```suggestion
         * Gets or sets data in an "Addin" field. If the field isn't an "Addin" field, it is null and it will throw a general exception when code attempst to set it.
```

Reply:
Updated